eppo/src/main/java/cloud/eppo/android/dto/adapters/EppoValueAdapter.java

```diff
@@ -45,9 +45,6 @@ public EppoValue deserialize(JsonElement json, Type typeOfT, JsonDeserialization
 
             try {
                 String stringValue = json.getAsString();
-                if (stringValue == "null") {
-                    return EppoValue.valueOf();
-                }
```

Comment:
String equality uses `.equals()` so this was never true and we don't even do this shenanigans in other SDKs / UFC so just axing it